manifests/instance.pp

```diff
@@ -210,9 +216,18 @@
     $redis_file_name_orig = $config_file_orig
     $redis_file_name      = $config_file
   } else {
-    $redis_config_extension    = ".${title}"
-    $redis_file_name_orig      = "${config_file_orig}${redis_config_extension}"
-    $redis_file_name           = "${config_file}${redis_config_extension}"
+    $redis_server_name    = "redis-server-${name}"
```

Comment:
If we wanted to be really pedandtic we could change it to be `redis` on RedHat and `redis-server` on Debian to follow convention, but I don't care that much